data/project/emberData/beta.md

```diff
@@ -4,11 +4,11 @@ baseFileName: ember-data
 filter:
  - /ember-data\./
 repo: emberjs/data
-lastRelease: 3.16.0-beta.0
-futureVersion: 3.16.0-beta.0
-finalVersion: 3.16.0
+lastRelease: 3.17.0-beta.0
+futureVersion: 3.17.0-beta.1
+finalVersion: 3.17.0
 channel: beta
-date: 2019-12-20
+date: 2020-02-18
```

Comment:
According to https://libraries.io/npm/ember-data the release is dated January 25th.

```suggestion
date: 2020-01-25
```